### PR TITLE
Fix: Changed US-03 backend test to use a valid day. Included dotenv in knexfile.js

### PR DESCRIPTION
**Pull request by gabrielsanchez**

### CODE REVIEW

**Comment by gabrielsanchez** [comment]
@dalelotts does this look good to you? I changed the date from 2050-01-04 to 2050-01-05 because 2050-01-04 falls on a Tuesday and it would ignore the tests. I also included the dotenv import in the knexfile.

**Comment by dalelotts** [comment]
Can you tell me a bit about which problem this fixes?

**Comment by dalelotts** [comment]
Oh, I see. The changes look good based on your description of problem.  

Unrelated to this change,  that test should be broken up into multiple tests, I'm not sure why the author decided to write one test for multiple conditions.

File: back-end/test/us-03-create-reservations-eligible-timeframe.test.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ describe("US-03 - Create reservations eligible timeframe", () => {
         .send({ data });
       expect(response.status).toBe(400);
 
-      data.reservation_time = "15:30";
+      data.reservation_time = "05:30";
```
Comment (dalelotts) [inline back-end/test/us-03-create-reservations-eligible-timeframe.test.js:53]: Why was it necessary to change the time?

Comment (gabrielsanchez) [reply back-end/test/us-03-create-reservations-eligible-timeframe.test.js:53]: the restaurant is open at 15:30. It should either be a different hour or return a 200.

**Comment by dalelotts** [comment]
LGTM

**Pull request merged**